subjects/devops/easy-perm-checkpoint/README.md

```diff
@@ -0,0 +1,29 @@
+## easy-perm
+
+> 🕵️ **Operation Data Centre**
+>
+> [Confidential]
+>
+> You are on a secret mission inside an underground data centre. You are alone, there is no internet access and there is no mobile phone signal.
+>
+> You cannot interact with anyone.
+>
+> You cannot not access the internet - although you can read the mission below.
+>
+> You may access man pages via the command line. This is your only resource.
+>
+> Good luck agent, we're counting on your knowledge.
+
+### Instructions
+
+Create a file `easy-perm.sh`, which will change the default permissions for the `example.txt` and `example2.txt` files inside the folder `easy-perms`, to the ones bellow:
```

Comment:
below is with a single `l`

Reply:
Nice one for spotting it. I actually never read them - I just removed the hints. But it seems the typos exists in man places. I'll remove them all in a separate commit.